host/bcm283x/gpio.go

```diff
@@ -302,7 +305,8 @@ func (p *Pin) Halt() error {
 // PWM1 is exposed on pins 13, 19, 41 and 45.
 //
 // PWM0 and PWM1 share the same clock source, so even if they can have
-// independent duty cycle, they must have the same period.
+// independent duty cycle, they must have the same period. Pins shared on one
```

Comment:
No, they don't need to have the same period if the are on different PWM channels.
For example, you can set different periods for GPIO 12 and 13 in your sample code.

I think what you call "oversampling" is already there.
The clock is fixed to 25MHz and we specify the periods by rng1 and rng2 registers as well as duties are specified by dat1 and dat2 independently.

Reply:
True, but it used to not be the case, fixed the doc.